contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -443,6 +443,13 @@ NXmpes(NXobject):
             doc: |
               In the case of an experiment in which the electron current is changed and
               recorded with time stamps, this is an array of length m of current setpoints.
+    instrument_history(NXhistory):
+      exists: optional
+      doc: |
+        A set of activities that occurred to the instrument prior to/during the photoemission
+        experiment, including any activities performed on the individual instrument parts.
+        This group can be used to describe the preparation of the instrument prior to the
+        experiment, e.g. the cleaning procedure for a spin filter crystal.
```

Comment:
Maybe that level is also sufficient, but it would lead to various information being not at the "right" place, I think.